newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js

```diff
@@ -19,12 +19,12 @@ const editors: Array<ResourceExternalEditor> = [
     },
   },
   {
-    name: 'Jsfx',
-    displayName: 'Create a New Sound effect with Jsxf (*.wav)',
+    name: 'Jfxr',
+    displayName: 'Create/Edit a Sound effect with Jfxr (*.wav)',
```

Comment:
Create/Edit is better indeed :)

Reply:
It better communicates to the user that gdevelop actually stores the sound effects metadata :)